git-keeper-client/tests/test_client_configuration.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2016 Nathan Sommer and Ben Coleman
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import gkeepclient.client_configuration as client_config
+
+
+def test_config_values():
```

Comment:
Thinking toward UX:  what does the exception look like when things go wrong?  How hard will it be to translate this into  a human-friendly message?

Is this testable?  I'm not sure.